.github/workflows/cmake-embedded.yml

```diff
@@ -89,7 +89,7 @@ jobs:
         tag: ${{steps.gitversion.outputs.major}}.${{steps.gitversion.outputs.minor}}.${{steps.gitversion.outputs.patch}}+${{steps.gitversion.outputs.build}}-${{ steps.branch-name.outputs.current_branch }}
 
     - name: Delete dev Branch
-      if: ${{ startsWith(github.ref, 'refs/heads/dev') }}
+      if: startsWith(github.ref, 'refs/heads/dev')
       run: |
         git checkout -b temp \ 
```

Comment:
I don't think you need this checkout step

Reply:
How do you suggest getting branch name?

Reply:
Does:
```yaml
    - name: Get branch name
       id: branch-name
       uses: tj-actions/branch-names@v5.1
```
not work?

Reply:
oh nevermind. the GitHub app was displaying this comment on the wrong line.